Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/06-AKS-supporting/supporting-infra.tf

```diff
@@ -32,3 +32,20 @@ module "create_kv" {
   zone_resource_group_name = data.terraform_remote_state.existing-lz.outputs.lz_rg_name
 
 }
+
```

Comment:
Please add comment indicating that if Blue/Green scenario is not used then a public DNS zone is not needed and this code can be removed.

Reply:
fixed